test/e2e/integreatly_test.go

```diff
@@ -445,11 +442,11 @@ func checkIntegreatlyNamespaceLabels(t *testing.T, f *framework.Framework, names
 		namespace := &corev1.Namespace{}
 		err := f.Client.Get(goctx.TODO(), client.ObjectKey{Name: intlyNamespacePrefix + namespaceName}, namespace)
 		if err != nil {
-			return errors.Wrap(err, "Error getting namespace: "+namespaceName+" from cluster")
+			return fmt.Errorf("Error getting namespace: %v from cluster: %w", namespaceName, err)
 		}
 		value, ok := namespace.Labels[label]
 		if !ok || value != "true" {
-			return errors.Wrap(err, "Incorrect label on integreatly namespace: "+namespaceName+". Expected: "+label+". Got: "+label+"="+value)
+			return fmt.Errorf("Incorrect %v label on integreatly namespace: %v. Expected: true. Got: %v. %w", label, namespaceName, value, err)
```

Comment:
@grdryn similiar issue here^, `err` object is irrelevant to this error statement